### PR TITLE
Fixed NavBar Style

### LINKED ISSUE #3071: Unexpected Behaviour of NavBar [UI-issue]
### Describe the bug

When Opening more than 5 tabs (locked) there is a horizontal scroll bar and after scrolling the Navbar is not Sticked.
image

### To Reproduce

1. Open more than 5 tabs
2. Lock all the tabs 
3. Scroll side to the end 
4. See error there is no Navbar 

### Expected behavior
Either Navbar should be sticked. OR We should limit users to 5 tabs

### Additional Context
I would like to work on this 😃

### PR DESCRIPTION
**Pull request by yashrajbothra**
Fixes #3071

### CODE REVIEW

File: interface/main/tabs/main.php [hunk 1]
```diff
@@ -331,7 +331,7 @@ function setupI18n(lang_id) {
 </style>
 
 </head>
-<body data-bind="css: responsiveDisplay.objWidth().bodyMain">
+<body data-bind="css: responsiveDisplay.objWidth().bodyMain" style="width: max-content;min-width: 100%;">
```
Comment (im-Amitto) [inline interface/main/tabs/main.php]: `max-content` is not supported in IE and `min-width: 100%` can be replaced with bs4 `mw-100`

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: Thanks for the review @im-Amitto

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: Btw @bradymiller is OpenEMR is supported in IE
image
I thought to test my changes and saw few million bugs 🤔

Comment (bradymiller) [reply interface/main/tabs/main.php]: This is a good question:
@sjpadgett , @tywrenn ; I recall this was discussed before. What IE version compatibility are we now shooting for?

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: I was on IE 11 and btw new Edge chromium has no compatiblity issue 👍

Comment (tywrenn) [reply interface/main/tabs/main.php]: @bradymiller @im-Amitto Internet Explorer is a dead browser in which compatibility is basically no longer possible with a lot of frameworks outright not supporting it. It also is actually now considered a security risk as it is not really maintained by Microsoft and is expected to be removed in the future. Therefore this next OpenEMR revision is basically not gonna support IE and should not support it.

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: RIP Internet Explorer. But i will never miss you🤣

Comment (bradymiller) [reply interface/main/tabs/main.php]: farewell my `not` friend, you will be truly `not` missed :)

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: mw-width is replacement for max-width so can't go for it 😞. Should i go for min-vw-100 🤔

Comment (tywrenn) [reply interface/main/tabs/main.php]: @yashrajbothra I don't think min-vw-100 even exists in BS4 lol

Comment (yashrajbothra) [reply interface/main/tabs/main.php]: https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-viewport

Comment (tywrenn) [reply interface/main/tabs/main.php]: @yashrajbothra guess I learn something new everyday 😂

Comment (im-Amitto) [reply interface/main/tabs/main.php]: @yashrajbothra  Sorry, I read it `max-width`. I guess i need to re-stock my caffine :confounded:

**Comment by bradymiller** [comment]
I just tested this PR and having following issue (right half screen is white):

image

It reverts back to normal if I remove the changes in this PR. I'm using Firefox 73.0.1.

**Comment by yashrajbothra** [comment]
Sorry, My bad i should have test before commiting 😞.

**Comment by bradymiller** [comment]
hi @yashrajbothra , no prob. now is testing well and will bring into codebase. Thanks for the fix! -brady

**Pull request merged**